#306 Member does not see the description of the Userstory

SponsoredByPuma opened the pull request:
Its currently working, but there are still some error in the console, because of the changed key of the v-for iteration.
We probably could fix this by giving the userstory another attribute e.g. position, so we could iterate with this.
Here is a screenshot of the error message: 
![image](https://github.com/Sybit-Education/Diveni/assets/92574150/38b987f0-95b0-4daf-a5b5-d1e52659e6ec)

KzuDemEvin commented on frontend/src/components/MobileStoryList.vue:
```diff
@@ -2,7 +2,7 @@
     <b-list-group-item
       v-for="(story, index) of userStories"
-      :key="story.name"
+      :key="story.position"
```
Why don't you use the property `id` of userStories or use the `index` as key?

KzuDemEvin commented on frontend/src/components/MobileStoryTitle.vue:
```diff
@@ -4,7 +4,7 @@
         v-for="(story, idx) of userStories"
         v-show="story.isActive"
-        :key="story.name"
+        :key="story.position"
```
Why don't you use the property `id` of userStories or use the index `idx` as key?

KzuDemEvin commented on frontend/src/components/UserStoryDescriptions.vue:
```diff
@@ -4,7 +4,7 @@
         v-for="(story, idx) of userStories"
         v-show="idx === index"
-        :key="story.name"
+        :key="story.position"
```
Why don't you use the property `id` of userStories or use the index `idx` as key?

KzuDemEvin commented on frontend/src/components/UserStoryDescriptions.vue:
```diff
@@ -95,43 +99,50 @@ export default Vue.extend({
   methods: {
+    valueChanged(idx, {markdown}) {
+      this.userStories[idx].description = markdown;
+    },
```
Perhaps you can already call `this.publishChanges(index);` here, so there is no need to press the button to update the user story for all members.

KzuDemEvin reviewed (changes requested):
- Is the position really needed as a property? Or can it be replaced by the id of the user stories or the index in the loop?
- I had a suggestion regarding the update policy for user stories.

SponsoredByPuma commented:
Thanks for the advice @KzuDemEvin. I removed the position attribute and added id as the key. Also I extended the mobile classes for the id attribute since it was missing in those. Last but not least the description now updates as soon as you click next to the markdown editor box.
